### DISCUSSION TITLE
Please add SF2 support back

### DISCUSSION THREAD

**Discussion by akamakakakakaalaka** [spessasus/SpessaSynth#45] [General]
I can’t use SF2 Files, I can only use DLS ones, Why? Bring SF2 Support back

**Comment by spessasus** [comment]
Erm, what? I can load SF2 files just fine. Are you sure you're using the `loadSoundFont` function?

**Comment by spessasus** [comment]
Also remember to update `worklet_processor.min.js` in your code to the latest version.

**Comment by akamakakakakaalaka** [comment]
But I am working on my iPhone 14! Supports the DLS! No SF2 files!

**Reply by spessasus** [reply]
I don't own any Apple devices, so I can't test your claims. What do you mean by "not working"? What kind of error are you getting?

The website works fine under both Chrome and Firefox on my Android device.

**Reply by akamakakakakaalaka** [reply]
I am running safari for my iPhone 14 Pro Max it doesn’t support SF2 only DLS, but I can import SF2s on gradma’s iMac

**Reply by spessasus** [reply]
Then it seems like your iPhone is the problem here. But you still didn't explain what you mean by "not supporting" sf2. The device itself doesn't have to support sf2 files, spessa handles everything by itself. For example my POCO F5 doesn't support SF2, yet I can still import them on the website.
